tests/signing/testdata/signing/TestSigningPayload/sample_model_file

```diff
@@ -2,17 +2,25 @@
   "_type": "https://in-toto.io/Statement/v1",
   "subject": [
     {
-      "name": ".",
+      "name": "file_model",
       "digest": {
-        "sha256": "3aab065c7181a173b5dd9e9d32a9f79923440b413be1e1ffcdba26a7365f719b"
-      },
-      "annotations": {
-        "actual_hash_algorithm": "sha256"
+        "sha256": "002d162f867c5eee944e5080d25829b6625be0e3f081f6fbafc7dd655ca2e178"
       }
     }
   ],
-  "predicateType": "https://model_signing/Digests/v0.1",
+  "predicateType": "https://model_signing/signature/v1.0",
   "predicate": {
-    "unused": "Unused, just passed due to API requirements"
+    "serialization": {
+      "method": "files",
+      "hash_type": "sha256",
+      "allow_symlinks": true
+    },
+    "resources": [
+      {
+        "algorithm": "sha256",
+        "digest": "3aab065c7181a173b5dd9e9d32a9f79923440b413be1e1ffcdba26a7365f719b",
+        "name": "."
```

Comment:
this has to do with the fact that it's a single file model right? And it's relative path to the root? hence the `.` name?

Reply:
That's true.